src/components/LanguageSelector/index.less

```diff
@@ -0,0 +1,9 @@
+.language-select {
+  display: flex;
+  justify-content: center;
```

Comment:
Is this really needed?

Reply:
Indeed, justify-content can be removed. But display: flex is needed. If it is missing, the drop down-menu is not completely within the toolbox.